#,Module,TestFunction,Result,TimeTaken,ExecutedOn,Syslogs,FCLI,TSSH,DCNT,Description,Devices,KnownIssue,Doc
1,routing/BGP/test_bgp_4node.py,,,0:01:48,2024-01-07 22:40:18,0,1,0,4,Module Prolog,"V1, V2, V3, V4",,
2,routing/BGP/test_bgp_4node.py,TestBGPConfed.test_ipv6_confed_route_distribution,Fail,0:00:22,2024-01-07 22:40:34,0,1,0,4,Fail: Route advertised by DUT1 not found on DUT3,"V1, V2, V3, V4",,
3,routing/BGP/test_bgp_4node.py,TestBGPConfed.test_ipv6_confed_with_rr,Fail,0:00:48,2024-01-07 22:41:21,0,1,0,4,Fail: DUT4 did not learn the routes advertised by peer DUT2,"V1, V2, V3, V4",,
4,routing/BGP/test_bgp_4node.py,TestBGPConfed.test_confed_route_distribution_with_rmap,Pass,0:00:39,2024-01-07 22:42:00,0,1,0,4,Test case passed,"V1, V2, V3, V4",,
5,routing/BGP/test_bgp_4node.py,test_ft_bgp_ibgp_RR_Loop,ScriptError,0:00:39,2024-01-07 22:42:44,0,1,0,4,Exception IndexError list index out of range in file /tests/routing/BGP/test_bgp_4node.py at line 690,"V1, V2, V3, V4",,Verify the functioning of iBGP Route-Reflector cluster loop
6,routing/BGP/test_bgp_4node.py,test_ft_bgp_ebgp_community_map,Pass,0:04:21,2024-01-07 22:47:05,0,1,0,4,Test case passed,"V1, V2, V3, V4",,Verify the functioning of eBGP communities
7,routing/BGP/test_bgp_4node.py,,,0:00:36,2024-01-07 22:47:40,0,1,0,4,Module Epilog,"V1, V2, V3, V4",,
